http-server/src/main/java/io/micronaut/http/server/exceptions/response/HtmlErrorResponseBodyProvider.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017-2024 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.exceptions.response;
+
+import io.micronaut.context.annotation.DefaultImplementation;
+import io.micronaut.http.MediaType;
+
+/**
+ * A {@link ErrorResponseBodyProvider} for HTML responses.
+ * Responses with content type {@link io.micronaut.http.MediaType#TEXT_HTML}.
+ * @author Sergio del Amo
+ * @since 4.7.0
+ * @param <T> The body type
+ */
+@DefaultImplementation(DefaultHtmlErrorResponseBodyProvider.class)
+public interface HtmlErrorResponseBodyProvider<T> extends ErrorResponseBodyProvider<T> {
```

Comment:
Maybe T should be a String?

Reply:
resolved via https://github.com/micronaut-projects/micronaut-core/pull/11210/commits/6918fc35ca248f1b0e39f46649e8477c07f91553